Nexogen.Libraries.Metrics.Grpc/MetricsBase.cs

```diff
@@ -90,7 +90,10 @@ public void StreamMsgSent(MethodType type, string method)
 
         private static string[] GetLabels(MethodType type, string method)
         {
-            var split = method.Split('/');
+            var split = method.Split('.');
+            string serviceName = string.Join(".", split.Take(split.Length - 1));
```

Comment:
Would you please add a unit test for this logic, with real world examples? I think that would be a valuable.